main_routers/characters_router.py

```
@@ -146,6 +154,197 @@ async def send_reload_page_notice(session, message_text: str = "语音已更新
+        except Exception as exc:
+            rollback_errors.append(f"tombstones restore failed: {exc}")
+
+    try:
+        initialize_character_data = get_initialize_character_data()
+        await initialize_character_data()
+    except Exception as exc:
+        rollback_errors.append(f"initialize_character_data failed: {exc}")
+
+    try:
+        await notify_memory_server_reload(reason=reason)
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)